### PR TITLE
[patch] Enhancements for Event Streams support

### PR DESCRIPTION
**Pull request by andrercm**
Adds new parameters for the following:

- Event Streams support if we don't want defaulted AMQ Kafka
- Data Dictionary channel/catalog source to be customized
- Manage base and secondary languages

### CODE REVIEW

**Comment by andrercm** [comment]
Testing is ongoing with this, so still need to remove that ansible tar.gz, when it's done I'll set this ready to review.

File: image/cli/mascli/templates/pipelinerun.yaml [hunk 1]
```diff
@@ -175,6 +190,12 @@ spec:
       value: '$MAS_CATALOG_SOURCE'
     - name: mas_catalog_version
       value: '$MAS_CATALOG_VERSION'
+    - name: mas_add_channel
+      value: '$MAS_ADD_CHANNEL'
+    - name: mas_add_catalog
+      value: '$MAS_ADD_CATALOG'
+    - name: mas_catalog_version
```
Comment (whitfiea) [inline image/cli/mascli/templates/pipelinerun.yaml]: duplicate of line 191-192 above

File: image/cli/mascli/templates/pipelinerun.yaml [hunk 2]
```diff
@@ -175,6 +190,12 @@ spec:
       value: '$MAS_CATALOG_SOURCE'
     - name: mas_catalog_version
       value: '$MAS_CATALOG_VERSION'
+    - name: mas_add_channel
+      value: '$MAS_ADD_CHANNEL'
+    - name: mas_add_catalog
```
Comment (whitfiea) [inline image/cli/mascli/templates/pipelinerun.yaml]: these mas_add_* are also added lower day rather than here

**Review by whitfiea — CHANGES_REQUESTED** [review]

File: image/cli/mascli/templates/pipelinerun.yaml (hunk 1 above)
Comment (andrercm) [reply image/cli/mascli/templates/pipelinerun.yaml]: oh thanks! i'll remove that and also:
```
    - name: mas_catalog_version
      value: '$MAS_CATALOG_VERSION'
```

which is also duplicate of line 191

File: image/cli/mascli/templates/pipelinerun.yaml (hunk 2 above)
Comment (andrercm) [reply image/cli/mascli/templates/pipelinerun.yaml]: Removed the dups from `/templates/pipelinerun.yml`

**Review by whitfiea — APPROVED** [review]

**Pull request merged**